.vscodeignore

```diff
@@ -6,3 +6,12 @@
 src/**
 out/test/**
 **/*.js.map
+SampleApplication/
+SampleExponentApplication/
```

Comment:
This was after talking with @MSLaguana. Basically we don't want to ship our Samples with the extension.